angr/analyses/propagator/propagator.py

```diff
@@ -376,6 +378,19 @@ def _initial_abstract_state(self, node):
                                  self.project.arch.bytes,
                                  SpOffset(self.project.arch.bits, 0)
                                  )
+            if self._initial_registers:
+                for reg, val in self._initial_registers.items():
+                    if val is not None:
+                        reg_offset, reg_size = self.project.arch.registers[reg]
+                        state.store_register(reg_offset, reg_size, val)
+            ## Use the registers from the base state
+            #if self._base_state:
+            #    for reg_name in dir(self._base_state.regs):
+            #        offset, size = self.project.arch.regsters[reg_name]
+            #        reg_val = getattr(self._base_state.regs, reg_name)
+            #        if reg_val.concrete:
+            #            real_regval = self._base_state.solver.eval_one(reg_val)
+            #            state.store_register(offset, size, real_regval)
```

Comment:
I suppose the above commented out lines don't matter any more. Can we get rid of them?